benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/explorationvalues/IntValuesTest.scala

```diff
@@ -50,4 +50,92 @@ class IntValuesTest extends JUnitSuite {
 
   }
 
```

Comment:
I added now more tests for the cases you specified. I also added subtraction so that we also can handle decreasing sequences.